persistence-modules/spring-boot-persistence-2/README.md

```diff
@@ -1,3 +1,3 @@
 ### Relevant Articles:
 
-- [Using JDBI with Spring Boot](https://www.baeldung.com/spring-boot-jdbi)
+- [Using JDBI with Spring Boot](https://www.baeldung.com/spring-boot-jdbi)
```

Comment:
Let's remove this file from your PR (no updates needed)

Reply:
I've replaced the README.md with the master version.